package.json

```diff
@@ -34,7 +34,7 @@
     "test:cover": "karma start build/karma.cover.config.js",
     "test:e2e": "npm run build -- vue.min.js && node test/e2e/runner.js",
     "http-server": "http-server ./ -s",
-    "testcafe": "testcafe chrome test/e2e/tests/*.js",
+    "testcafe": "testcafe chrome, firefox test/e2e/tests/*.js",
```

Comment:
it doesn't work because a space symbol between `chrome,` and `firefox`. 
should be
`testcafe chrome,firefox test/e2e/tests/*.js`